logic/broadcaster.js

```diff
@@ -115,6 +118,12 @@ Broadcaster.prototype.getPeers = function (params, cb) {
 			self.consensus = consensus;
 		}
 
+		if (params.broadhash && (params.matchBroadhash || params.unmatchBroadhash)) {
```

Comment:
If you want to get peers with unmatched broadhash - if will not work. It's because of how `modules.peers.list` works. When `broadhash` parameter is present - it will try to get peers with matched broadhash first, then only if amount of matched peers is lower than limit - if will get peers with unmatched broadhash, but up to limit. So when there is 100 peers with matched broadhash - you will only get those, if there is 99 peers with matched broadhash - you will get 99 matched + 1 unmatched.